scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/core/properties/PASchedulerProperties.java

```diff
@@ -352,7 +352,9 @@ negative number indicates that session is infinite (value specified in second) *
     SCHEDULER_DB_RECOVERY_LOAD_JOBS_BATCH_SIZE(
             "pa.scheduler.db.recovery.load.jobs.batch_size",
             PropertyType.INTEGER,
-            "100"),
+            "200"),
```

Comment:
Unsure why you update this value in this commit.
I think there is no relation

Reply:
yes wrong update